irs-api/src/main/java/org/eclipse/tractusx/irs/aaswrapper/job/delegate/DigitalTwinDelegate.java

```diff
@@ -98,7 +97,38 @@ public ItemContainer process(final ItemContainer.ItemContainerBuilder itemContai
         return itemContainerBuilder.build();
     }
 
-    private Tombstone createNoBpnProvidedTombstone(final JobParameter jobData, final PartChainIdentificationKey itemId) {
+    private void createShellEndpointCouldNotBeRetrievedTombstone(
+            final ItemContainer.ItemContainerBuilder itemContainerBuilder, final PartChainIdentificationKey itemId,
+            final Exception exception) {
+
+        // TODO (mfischer) is this log message and method name correct?
```

Comment:
open TODO

Reply:
@ds-jhartmann yes, we should discuss this, I am not sure here

Reply:
resolved